book/http_fundamentals.rst

```diff
@@ -41,9 +41,6 @@ Symfony is built from the ground up around that reality. Whether you realize
 it or not, HTTP is something you use every day. With Symfony, you'll learn
 how to master it.
 
-.. index::
-   single: HTTP; Request-response paradigm
```

Comment:
Well page is something I never came across :) Thank you! Clarified a few things, yes.